.github/workflows/CI.yml

```diff
@@ -28,6 +28,27 @@ jobs:
       - name: lint
         run: npm run lint
 
+  test_bundle:
+    name: Test production
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 14
```

Comment:
Are we still targeting node 14? Should we use the latest? Or maybe a matrix?

Reply:
this one is a good question @fvictorio

Reply:
For now we will rely on the V8 virtual machine to run the tests in an isolated context.
We are using node 14 because that's the node version that will build our production code.

Reply:
Agree, this seems enough. We are using a version matrix for the `test_linux` job, which should be enough (we use the same strategy in Hardhat).